docs/operations.rst

```diff
@@ -0,0 +1,11 @@
+Camus Operations
+==========================
+
+In this document, you will learn how to use Camus in production.
+
+.. toctree::
+   :maxdepth: 3
+
+   monitoring
```

Comment:
Asked @junrao to reach to Ken and Felix on monitoring and operation suggestions.